CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.5.0 (2022-06-23)
+
+- ???
```

Comment:
My change last week:

* Change "fromSubAccountId: string" param to "fromSubAccount: number[]".